bin/cli.js

```diff
@@ -149,6 +151,17 @@ const setRequestHeaders = () => {
 };
 
 const loadConfigMiddleware = async options => {
+  const validateConfig = () => {
+    const shouldValidate = options._.every(
+      item => !NO_VALIDATE_LIST.includes(item)
```

Comment:
`NO_VALIDATE_LIST ` is only used in this file so I don't think it needs to live in constants.ts. That file is kind of a messy grab-bag of stuff. 

Looking at this list:
```
  'init',
  'clean',
  'list',
  'remove',
  'use',
  'delete',
```

Is that the list of commands that currently aren't running config validation? I wonder if we could remove every command from that list except for `init`. All of the other commands rely on a valid config existing in order to function properly.

Reply:
Agreed! I think the only other command to add to the list is `auth`, since we want to allow folks to add an authorized account to an empty config.

Reply:
Addressed in [24c78e2](https://github.com/HubSpot/hubspot-cli/pull/1307/commits/24c78e289fc7bf4e8832826b131a7ce821d8c462)